docs/integrations/destinations/astra.md

```diff
@@ -0,0 +1,48 @@
+# Astra Destination
+
+This page contains the setup guide and reference information for the destination-astra connector.
+## Pre-Requisites
+
+- An OpenAI, AzureOpenAI, Cohere, etc. API Key
+
+## Setup Guide
+
+#### Set Up an Astra Database
+
+- Create an Astra account [here](https://astra.datastax.com/signup)
+- In the Astra Portal, select Databases in the main navigation.
+- Click Create Database.
+- In the Create Database dialog, select the Serverless (Vector) deployment type.
+- In the Configuration section, enter a name for the new database in the Database name field.
+-- Because database names can’t be changed later, it’s best to name your database something meaningful. Database names must start and end with an alphanumeric character, and may contain the following special characters: & + - _ ( ) < > . , @.
+- Select your preferred Provider and Region.
+-- You can select from a limited number of regions if you’re on the Free plan. Regions with a lock icon require that you upgrade to a Pay As You Go plan.
+- Click Create Database.
+-- You are redirected to your new database’s Overview screen. Your database starts in Pending status before transitioning to Initializing. You’ll receive a notification once your database is initialized.
+
+#### Setting up a Vector Collection
```

Comment:
Do we need this section with the new logic of the connector auto-creating the collection? If there's a good reason to still do it manually in some cases we can leave it in with a note that the connector can also take care of it, wdyt?

Reply:
Removed